_examples/range/range.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
It's using the built-in middleware layer, as multiplexer. 

Not sure what you're asking here...